.gitignore

```diff
@@ -7,3 +7,5 @@ Gemfile.lock
 /internal/awstesting/integration/smoke/_test/
 /vendor
 /private/model/cli/gen-api/gen-api
+.idea/
+*.DS_Store
```

Comment:
Would prefer not to include these in the SDK's `.gitignore` file you. For Git you can configure your local environment to ignore these platform/IDE files via git's global exclude configuration options.

https://help.github.com/en/articles/ignoring-files